BREAKING.md

```diff
@@ -329,6 +543,10 @@ The `width` property has been removed in favor of using CSS styling.
   </ion-split-pane>
   ```
 
+#### Tabs
+
+- [Focused State](#activated-focused-hover-states) have been updated.
```

Comment:
```suggestion
- [Focused States](#activated-focused-hover-states) have been updated.
```

Reply:
It's just one state for focused

Reply:
`have` should probably say `has` in that case